test/metalearn/metafeatures/test_metafeatures.py

```diff
@@ -60,7 +62,7 @@ def tearDown(self):
 
     def test_run_without_fail(self):
         for filename, dataframe in self.dataframes.items():
-            metafeatures_df = Metafeatures().compute(dataframe)
+            metafeatures_df = Metafeatures().compute(X=dataframe[0],Y=dataframe[1])
```

Comment:
Instead of using a tuple, consider using a nested dictionary like
```
{
  "X": X,
  "Y": Y
}
```
when the self.dataframes is created.